src/pl/plpython3/Makefile

```diff
@@ -153,6 +153,8 @@ REGRESS := $(foreach test,$(REGRESS),$(if $(filter $(test),$(REGRESS_PLPYTHON3_M
 .PHONY: pgregress-python3-mangle
 pgregress-python3-mangle:
 	$(MKDIR_P) sql/python3 expected/python3 results/python3
+	# for python and python3 types and returns test are not the same.
+	cp -f ../plpython/expected/plpython_returns_3.out ../plpython/expected/plpython_returns.out
```

Comment:
I think this line is not needed since we have line 174?
Plus this will create a git diff which is not ideal:
```
❯ git status
On branch hy/plpython3
Your branch is up to date with 'beeender/hy/plpython3'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   ../../backend/utils/error/elog.c
        modified:   ../plpython/expected/plpython_returns.out
```